crates/cheatnet/tests/contracts/Scarb.lock

```diff
@@ -1,6 +0,0 @@
-# Code generated by scarb DO NOT EDIT.
-version = 1
-
-[[package]]
-name = "cheatnet_testing_contracts"
-version = "0.1.0"
```

Comment:
It got removed when clearing git cache and readding everything, shouldn't have been there in the first place